packages/axelar-wasm-std/src/threshold.rs

```diff
@@ -127,7 +129,7 @@ impl TryFrom<Threshold> for MajorityThreshold {
     type Error = Error;
 
     fn try_from(value: Threshold) -> Result<Self, Error> {
-        if value.numerator() <= value.denominator() / Uint64::from(2u64) {
+        if value.numerator() <= value.denominator().div(Uint64::from(2u64)) {
```

Comment:
what's the reason for this change? Does this get around the clippy warning?

Reply:
yes, I wanted to delegate the handling to the underlying implementation like I did with `FiniteAmount`, in this case is Uint64 which panics when there is an issue during the div, but now that I think about it, I think this is not a good idea with structs external to us like Uint64 because clippy won't be able to detect an issue if implementation changes